dashboard/config/regional_partners_school_districts.tsv

```diff
@@ -11550,62 +11550,119 @@ Southern Nevada Regional Professional Development Program	3200480
 Southern Nevada Regional Professional Development Program	3200510
 Southern Nevada Regional Professional Development Program	5900061
 Southern Nevada Regional Professional Development Program	5900153
+Code/Interactive	3600001	csp	August 14 - 18, 2017
 Code/Interactive	3600001
```

Comment:
Not having any context, it seems strange that there would be two rows with the same numerical ID.